tools/api-markdown-documenter/src/api-item-transforms/configuration/Configuration.ts

```diff
@@ -66,9 +89,22 @@ export interface ApiItemTransformationConfiguration
  */
 export interface ApiItemTransformationOptions
 	extends ApiItemTransformationConfigurationBase,
-		Partial<ApiItemTransformations>,
 		DocumentationSuiteOptions,
-		LoggingConfiguration {}
+		LoggingConfiguration {
+	/**
+	 * Optional overrides for the default transformations.
+	 */
+	readonly transformations?: Partial<ApiItemTransformations>;
+
+	/**
+	 * {@inheritDoc ApiItemTransformationConfiguration.defaultSectionLayout}
+	 */
+	readonly defaultSectionLayout?: (
```

Comment:
I plan to deduplicate this with the duplicate signature above in a future change.